savant/client/runner/source.py

```diff
@@ -144,6 +145,24 @@ def send(self, source: Frame, send_eos: bool = True) -> SourceResult:
 
         return result
 
+    def send_batch(self, source_id: str, content: Batch) -> SourceResult:
```

Comment:
Make internal methods `_send_frame` and `_send_batch`. In `send` and `send_iter` call either of them. That's needed to allow those methods (and `__call__`) to accept both `VideoFrame` and `VideoFrameBatch`.